peps/pep-0738.rst

```diff
@@ -107,17 +106,20 @@ are:
   linker (lld), and headers for all the system libraries.
 
   Binary compatibility between libraries compiled with different versions of the
-  NDK is generally very good, but for reproducbility it would be best for each
+  NDK is generally very good, but for reproducibility it would be best for each
   Python version to stick with one NDK version throughout its life. For Python
   3.13, this would be the current NDK long-term support version, r26.
 
   Each NDK version can be set to target any of a wide range of Android versions.
-  For example, NDK r26 supports API levels 21 to 34. However, binaries compiled
-  for an older Android version will usually keep on working indefinitely on
-  newer versions; exceptions to this rule are only made for security reasons.
+  For example, NDK r26 supports :ref:`API levels <738-os-versions>` 21 to 34.
+  However, binaries compiled for an older Android version will usually keep on
+  working indefinitely on newer versions; exceptions to this rule are only made
+  for security reasons.
 
 * Gradle is the tool used to build complete, deployable apps.
 
+.. (spacer)
```

Comment:
Hmm, I can see why you did this, but it results in two `<ul>`s of two and one `<li>`s respectively, instead of a single `<ul>` of three `<li>`s:

<img width="187" alt="image" src="https://github.com/python/peps/assets/1324225/4cdbac69-43ec-4ccc-b40d-29a71eb4db49">

For accessibility, it'd be better to have the markup reflecting semantics: one list of three items.

Does anyone know if there's a better way to do add space between bullets in Sphinx that doesn't affect the underlying markup in this way?

Reply:
I spent some time searching but I couldn't find one. Apparently Sphinx only adds `<p>` tags inside the `<li>` if the item contains multiple paragraphs.

Reply:
@AA-Turner Any Sphinx suggestions for this?

Reply:
https://github.com/readthedocs/sphinx_rtd_theme/issues/354 suggests that `.. rst-class:: open` is supposed to wrap every list item in `<p>` tags. But when I try it, it doesn't have any effect except adding a CSS class, which has no corresponding rules in any style sheet.

Could we work around this by adding a rule to the PEPs theme?

```css
ul.open li {
    margin-top: 1em;
    margin-bottom: 1em;
}
```

Reply:
Something like that could work, would you like to open a new PR so we can get a build preview and check it in isolation?

Reply:
OK, I think I've found a solution which will improve the rendering of many other PEPs at the same time. So I've removed the manual adjustments from this PR, and I'll create an independent PR with my idea.

Reply:
* https://github.com/python/peps/pull/3662